docker-compose.yml

```diff
@@ -29,6 +29,7 @@ services:
       - broker
       - db
   web:
+    user: $UID
```

Comment:
Instead of needing to export ``UID`` i suggest we leave the dev dockerfile running as root.